lib/feedback.js

```diff
@@ -311,6 +407,48 @@ function feedback(system) {
 
 }
 
+feedback.prototype.refreshFeedback = function(feedback) {
+	var self = this;
+
+	if (feedback.type !== undefined && feedback.instance_id !== undefined) {
+		if (self.feedback_definitions[feedback.instance_id] !== undefined && self.feedback_definitions[feedback.instance_id][feedback.type] !== undefined) {
+			let definition = self.feedback_definitions[feedback.instance_id][feedback.type];
+			// Run the subscribe function if needed
+			if (definition !== undefined && definition.subscribe !== undefined && typeof definition.subscribe == 'function') {
```

Comment:
Both `definition.subscribe` should be `definition.refresh`

Reply:
Yep ... didn't test with only a refresh definition